frontend/package.json

```diff
@@ -35,6 +35,7 @@
     "draft-js-import-html": "^1.4.1",
     "draftjs-filters": "^3.0.1",
     "draftjs-to-html": "0.9.1",
+    "enquirer": "^2.4.1",
```

Comment:
Can I check what is this for?

Reply:
It was a peer dependency that was blocking CI

Reply:
As discussed, will move it to dev dep